### ISSUE TITLE
Show template description on launch

### ISSUE THREAD

**Issue by tikoflano** [ansible/awx#7254]
##### ISSUE TYPE
 - Show template description on launch modal

##### SUMMARY
Would be great to be able to see the template's description on the launch modal (first page). This would allow us to show some usefull information to whom is running the playbook:

- What the playbook does. This is partially covered when using the Extended list view for the templates. But the formatting is not great and it gets lost with the extra information.
- Playbook vars usage. (Ok, this can be done with comments)
- Some steps that must be done before or after the playbook is done.
- Warnings
- Most important: playbook available tags and when to use them. This would help, at least for now, the problem described at https://github.com/ansible/awx/issues/1182

**Comment by wenottingham** [comment]
@trahman73 is this in the design for this iteration?

**Comment by trahman73** [comment]
@wenottingham This is a nice to have for 4.0... The idea is to show a launch preview as the design is here:
https://tower-mockups.testing.ansible.com/patternfly/templates/templates-prompt/

**Comment by mabashian** [comment]
Link #8249.  If the PF Wizard component supported a custom header then we could display the description under the resource name.  Alas, that component only supports a string in the title so we'd have to come up with a different solution.  The launch prompt is tricky because the first step _could_ be one of many different steps depending on whats prompted.  We could add a dedicated step for the description but I'd prefer to find a solution where we didn't have to add steps to the flow.

**Comment by trahman73** [comment]
@mabashian We can redesign the mockup to not pass the title in the header, but add it as the 1st item in the body details as "Name" if that helps.

**Comment by mabashian** [comment]
I was mistaken about this component.  They do support a `description` prop that will render under the title.  I think this would be the perfect place for us to put the description of the launching resource (JT/WFJT/etc).

http://patternfly-react.surge.sh/components/wizard#wizard

**Comment by mabashian** [comment]
See #10260
